docs/docs/working-with-Etsy-in-Gatsby.md

```diff
@@ -0,0 +1,59 @@
+---
+Title: Guide on working with Etsy in Gatsby
+---
+
+# Guide on working with Etsy in Gatsby
```

Comment:
```suggestion
```
No need to duplicate the title in the main content. Also, the title in the frontmatter is automatically an h1, so for accessibility purposes we would not want to have another h1 in this document.